mysqld_exporter.go

```diff
@@ -262,6 +264,13 @@ const (
 		  FROM information_schema.schemata
 		  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
 		`
+	queryResponseCheckQuery = `SELECT @@query_response_time_stats`
+	queryResponseTimeQuery = `
+		SELECT
+		    TIME, COUNT, TOTAL
+		  FROM information_schema.query_response_time
+		  WHERE TIME <> 'TOO LONG'
```

Comment:
All done except this one.
Parsing "TOO LONG" row produces the following unnecessary lines:
```
mysql_info_schema_query_response_time_count_bucket{le="0"} 9768
```
which is wrong because histogram's "le" == 0 due to "TOO LONG" as TIME field to float automatic conversion. Moreover, we don't know what is TOO LONG explicitly in numeric and we can't override histogram's +Inf to be super-precise.
Something similar about `mysql_info_schema_query_response_time_total{le="0"} 0`.